src/components/views/rooms/Autocomplete.js

```diff
@@ -41,6 +60,11 @@ export default class Autocomplete extends React.Component {
     }
 
     componentWillReceiveProps(newProps, state) {
+        if (this.props.room.roomId !== newProps.room.roomId) {
+            this.autocompleter.destroy();
+            this.autocompleter = new Autocompleter();
```

Comment:
I'm guessing this should have `newProps.room` going into it?

Reply:
ooh, good spot